### DISCUSSION TITLE
GitHub org for related repos?

### DISCUSSION THREAD

**Discussion by rdhelms** [levmason/flockpocket#5] [General]
@levmason Whenever I get a chance to work on a Vue front-end example, would it be helpful to have that grouped together with this repo under a `flockpocket` GitHub organization? I'm guessing that over time we may end up with a suite of different tools or examples for people to pull from in order to accomplish what they want to accomplish in the way that works best for them.

I'd be happy to help set that up if it would be useful, but if that's not appealing for any reason then I can just make the repo under my username.

Some GitHub org features are paid, but the free tier includes unlimited public and private repos.

**Comment by levmason** [comment]
Yeah that seems to make sense! I use bitbucket at work, so I'm not very familiar with github stuff, but this sounds like a "project"....just some hierarchy to organize multiple repos under a single container? Anyway, it sounds like a good idea. We'll probably have some android/iOS apps at some point as well. Also, were you suggesting to separate the webapp into it's own repo? At the time I thought you were just talking about the directory structure.

**Comment by rdhelms** [comment]
Yep, it's pretty analogous to a Bitbucket project.

And for the webapp, I could definitely do either (staying in a totally separate repo or a PR back to the current repo modifying the directory structure) - I just wanted to make sure to be out of the way of your work so that you could still keep making progress while also having the Vue example to look at.

**Comment by levmason** [comment]
Yeah a new repo probably makes sense for this. I'll have to spend some time figuring out the best way to separate the webapp from Django etc.
